src/SlackConnector/Connections/Sockets/Messages/Inbound/MessageInterpreter.cs

```diff
@@ -84,5 +87,20 @@ private static ChatMessage GetChatMessage(string json)
 
             return message;
         }
+
+        private static ReactionMessage GetReactionMessage(string json)
+        {
+            var message = JsonConvert.DeserializeObject<ReactionMessage>(json);
```

Comment:
Why are you parsing this object and then overriding the properties later? Can't this just be done in the de-serialise?